include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_weight_two_stage_xdl_cshuffle.hpp

```diff
@@ -728,11 +771,11 @@ struct DeviceGroupedConvBwdWeightTwoStage_Xdl_CShuffle
             const ADataType* p_a_grid = arg.p_a_grid_;
             const BDataType* p_b_grid = arg.p_b_grid_;
 
-            if constexpr(is_NGCHW_GKYXC_NGKHW<InLayout, WeiLayout, OutLayout>() ||
-                         is_NGCDHW_GKZYXC_NGKDHW<InLayout, WeiLayout, OutLayout>())
+            if constexpr(is_NGCHW_NGKHW<InLayout, WeiLayout, OutLayout>() ||
+                         is_NGCDHW_NGKDHW<InLayout, WeiLayout, OutLayout>())
             {
                 p_a_grid = type_convert<const ADataType*>(arg.p_workspace_) +
-                           arg.GetWorkspaceETensorSizeBytes() / sizeof(BDataType);
+                           arg.GetWorkspaceETensorSizeBytes() / sizeof(ADataType);
```

Comment:
I'm really confused with this. Above: 
https://github.com/ROCm/composable_kernel/pull/2023/files#diff-28e60dd58c796089a14d8b018b5b3e53797e46f9167e1668cc86c0f6e4dd90dbR691-R692
For this case you ask for workspace for all A/B/E tensors .

Reply:
We split Workspace on [EWorkspace, AWorkspace, BWorkspace]. This order is different because in two stage kernel we always need EWorkspace (so E is first in the memory).

Reply:
It would be **really** good to add a comment somwhere which would note this fact.